.github/CODEOWNERS

```diff
@@ -49,7 +49,7 @@
 /test/functional/apps/visualize/ @elastic/kibana-visualizations
 
 # Application Services
```

Comment:
Can we create such a section for GX?

- So the packages list at the bottom of the file, would still stay as is.
- But everything else, that is not a package, but is part of GX, would have `# Global Experience` section.